Bricks/net/tcp/test.cc

```diff
@@ -240,13 +241,17 @@ TEST(TCPTest, EchoLongMessageTestsDynamicBufferGrowth) {
 }
 
 // Don't run this ~1s test more than once in a loop.
-struct OnlyCheckForUnresolvableURLOnceSingleton {
+struct RunResolveAddressTestOnlyOnceSingleton {
   bool done = false;
 };
 
-TEST(TCPTest, ResolveAddressException) {
-  bool& done = Singleton<OnlyCheckForUnresolvableURLOnceSingleton>().done;
+TEST(TCPTest, ResolveAddress) {
+  bool& done = Singleton<RunResolveAddressTestOnlyOnceSingleton>().done;
   if (!done) {
+    EXPECT_EQ("127.0.0.1", ResolveIPFromHostName("localhost"));
```

Comment:
Nit: In *nix world, `n` is hardly capitalized in `Hostname`; it's a stable word by itself.